bblfsh-dashboard/README.md

```diff
@@ -0,0 +1,31 @@
+<img src="images/logo.png" width="100" align="right" vspace="20" />
+
+# Babelfish Dashboard
+
+This chart deploys a [dashboard](https://github.com/bblfsh/dashboard) for [Babelfish](https://doc.bblf.sh/)
+
+## Official Documentation
+
+Official project documentation found [here](https://doc.bblf.sh/)
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release bblfsh-dashboard
+```
+
+## Configuration
+
+See `values.yaml` for the available configuration parameters. Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`.
+
+For example, to used a already deployed bblfsh server we set the server address:
```

Comment:
s/used/use. Actually "to use an existing bblfsh server" sounds better. Also, I'd explain somewhere in the document that, if not provided, a local one will be set up.